config/default/manager_auth_proxy_patch.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: controller-manager
+  name: manager
```

Comment:
Why are we changing the name of the deployment?

Reply:
@VaishnaviHire you can find more context in https://github.com/operator-framework/operator-lifecycle-manager/issues/952#issuecomment-639657949. Changing the Deployment name is a way to avoid the OLM upgrade fails because these labels are immutable.

Reply:
Thanks @astefanutti !! Since we are changing the deployment name, we need to make sure we update the tests downstream as well.